### PR TITLE
add notes on installing with argocd

### PR DESCRIPTION
**Pull request by eddycharly**
This PR add notes on installing with argocd.

### CODE REVIEW

File: content/en/docs/Installation/_index.md [hunk 1]
```diff
@@ -55,6 +55,45 @@ To install the Kyverno [Pod Security Standard policies](/policies/pod-security/)
...
+
+When deploying this chart with ArgoCD you will need to enable `Replace` in the `syncOptions`, and you probably want to ignore diff in aggregated cluster roles.
+
+You can do so by following instructions in these pages of ArgoCD documentation:
+- [Enable Replace in the syncOptions](https://argo-cd.readthedocs.io/en/stable/user-guide/sync-options/#replace-resource-instead-of-applying-changes)
+- [Ignore diff in aggregated cluster roles](https://argo-cd.readthedocs.io/en/stable/user-guide/diffing/#ignoring-rbac-changes-made-by-aggregateroles)
+
+ArgoCD uses helm only for templating but applies the results with `kubectl`.
+
+Unfortunately `kubectl` adds metadata that will cross the limit allowed by Kuberrnetes. Using `Replace` overcomes this limitation.
```
Comment (chipzoller) [inline content/en/docs/Installation/_index.md:68]: "Kubernetes"

File: content/en/docs/Installation/_index.md [hunk 2]
```diff
@@ -55,6 +55,45 @@ To install the Kyverno [Pod Security Standard policies](/policies/pod-security/)
 helm install kyverno-policies kyverno/kyverno-policies -n kyverno
 ```
 
+### Notes for ArgoCD users
+
+When deploying this chart with ArgoCD you will need to enable `Replace` in the `syncOptions`, and you probably want to ignore diff in aggregated cluster roles.
```
Comment (chipzoller) [inline content/en/docs/Installation/_index.md:60]: The statement "and you probably want to ignore diff in aggregated cluster roles" needs more clarification. Even I'm not clear what this means and what I should do.

File: content/en/docs/Installation/_index.md [hunk 3]
```diff
@@ -55,6 +55,45 @@ To install the Kyverno [Pod Security Standard policies](/policies/pod-security/)
...
+
+You can do so by following instructions in these pages of ArgoCD documentation:
+- [Enable Replace in the syncOptions](https://argo-cd.readthedocs.io/en/stable/user-guide/sync-options/#replace-resource-instead-of-applying-changes)
+- [Ignore diff in aggregated cluster roles](https://argo-cd.readthedocs.io/en/stable/user-guide/diffing/#ignoring-rbac-changes-made-by-aggregateroles)
+
+ArgoCD uses helm only for templating but applies the results with `kubectl`.
```
Comment (chipzoller) [inline content/en/docs/Installation/_index.md:66]: Helm is a proper noun and is capitalized.

File: content/en/docs/Installation/_index.md [hunk 4]
```diff
@@ -55,6 +55,45 @@ To install the Kyverno [Pod Security Standard policies](/policies/pod-security/)
...
+
+Unfortunately `kubectl` adds metadata that will cross the limit allowed by Kuberrnetes. Using `Replace` overcomes this limitation.
+
+Another option is to use server side apply, this will be supported in ArgoCD v2.5.
```
Comment (chipzoller) [inline content/en/docs/Installation/_index.md:70]: "Another option is to use server-side apply, supported in ArgoCD v2.5+."

File: content/en/docs/Installation/_index.md [hunk 5]
```diff
@@ -55,6 +55,45 @@ To install the Kyverno [Pod Security Standard policies](/policies/pod-security/)
...
+
+Below is an example of ArgoCD application manifest that should work with this chart:
+```
```
Comment (chipzoller) [inline content/en/docs/Installation/_index.md:73]: Always add a new line surrounding code blocks. Language should also be specified, here it's `yaml`.

Comment (chipzoller) [reply content/en/docs/Installation/_index.md:73]: "Below is an example of an ArgoCD Application manifest that should work with the Kyverno Helm chart:"

**Review by chipzoller — CHANGES_REQUESTED** [review]

**Comment by eddycharly** [comment]
@chipzoller I addressed your comments.

**Review by chipzoller — APPROVED** [review]

**Pull request merged**